grammar.js

```diff
@@ -371,6 +372,16 @@ module.exports = grammar({
       )
     )),
 
+    ternary_expression: $ => prec.left(
+      seq(
+        field('condition', $._expression),
+        '?',
+        field('trueValue', $._expression),
```

Comment:
Usually we don't use camel case for these field names. How about the `consequence` (instead of `trueValue`) and `alternative` (instead of `falueValue`)? We use those names in some of the other Tree-sitter grammars

Reply:
Fixed 👍